### PR TITLE
python3Packages.ics: fix build

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Fix build

ZHF: #122042
Ref: https://hydra.nixos.org/build/142605104/nixlog/1

CC @NixOS/nixos-release-managers

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 122684` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.ics</li>
    <li>python39Packages.ics</li>
  </ul>
</details>

File: pkgs/development/python-modules/ics/default.nix [hunk 1]
```diff
@@ -9,21 +14,35 @@ buildPythonPackage rec {
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
-    owner = "C4ptainCrunch";
+    owner = "ics.py";
     repo = "ics.py";
```
Comment (Ma27) [inline pkgs/development/python-modules/ics/default.nix:18]: The upstream repo appears to be `github.com/ics-py/ics-py`

Comment (fabaff) [reply pkgs/development/python-modules/ics/default.nix]: I guess that it would help if I clean my :eyeglasses: 

Thanks, updated.

Comment (Ma27) [reply pkgs/development/python-modules/ics/default.nix]: Hehe, LGTM now :)

**Pull request merged**